api/core/services/customers/customer_service.py

```diff
@@ -24,6 +29,106 @@
 
 _logger = logging.getLogger(__name__)
 
+WORKFLOW_APP_URL = os.environ.get("WORKFLOWAI_APP_URL", "https://workflowai.com")
```

Comment:
that's already avaialble in `consts.py` no ?

Reply:
https://github.com/WorkflowAI/WorkflowAI/pull/161/commits/800989048491d04abd81701f255c01bfd02f466f

Reply:
yes thanks